File: modulo1/segmento3/apunte7/ejercicio4/ejercicio4.py
def tf(num1):

	if num1%2 == 0:
		respuesta = True
	else:
		respuesta = False
	
	return respuesta

num1 = int(input("Ingrese un numero: "))

respuesta= tf(num1)

if respuesta:
	print("Verdadero")

else:
	print("Falso")

File: modulo1/segmento3/apunte7/ejercicio4/ejercicio4.java
import java.util.Scanner;

public class ejercicio4 {
	public static boolean tf(Integer num1) {
		return num1 % 2 == 0;
	}
		
	
	public static void main(String args[]) {
		Scanner teclado = new Scanner(System.in);

		System.out.println("Ingrese un numero: ");
		Integer num1 = Integer.parseInt(teclado.nextLine());

		System.out.println(tf(num1));
	}
}

